flexmeasures/data/models/generic_assets.py

```diff
@@ -427,34 +431,51 @@ def search_beliefs(
         return bdf_dict
 
     @property
-    def sensors_to_show(self) -> List["Sensor"]:  # noqa F821
+    def sensors_to_show(self) -> list["Sensor" | list["Sensor"]]:  # noqa F821
         """Sensors to show, as defined by the sensors_to_show attribute.
 
         Sensors to show are defined as a list of sensor ids, which
         is set by the "sensors_to_show" field of the asset's "attributes" column.
         Valid sensors either belong to the asset itself, to other assets in the same account,
         or to public assets.
+        In case the field is missing, defaults to two of the asset's sensors.
+
+        Sensor ids can be nested to denote that sensors should be 'shown together',
+        for example, layered rather than vertically concatenated.
+        How to interpret 'shown together' is technically left up to the function returning chart specs,
+        as are any restrictions regarding what sensors can be shown together, such as:
+        - whether they should share the same unit
+        - whether they should share the same name
+        - whether they should belong to different assets
 
+        For example, this denotes showing sensors 42 and 44 together:
+
+            sensors_to_show = [40, 35, 41, [42, 44], 43, 45]
 
-        Defaults to two of the asset's sensors.
         """
         if not self.has_attribute("sensors_to_show"):
             return self.sensors[:2]
 
         from flexmeasures.data.services.sensors import get_sensors
 
-        sensor_ids = self.get_attribute("sensors_to_show")
+        sensor_ids_to_show = self.get_attribute("sensors_to_show")
         sensor_map = {
             sensor.id: sensor
             for sensor in get_sensors(
                 account=self.owner,
                 include_public_assets=True,
-                sensor_id_allowlist=sensor_ids,
+                sensor_id_allowlist=flatten_unique(sensor_ids_to_show),
```

Comment:
`flatten_unique` only allows one level of nesting. If there were two, we'd probably break here ― in a way that is difficult to understand.
Can we break in a better place, e.g. when `sensors_to_show` is being applied?

Reply:
I only meant to support `sensors_to_show` with one level of nesting. 1st level represents vertically concatenated charts, 2nd level represents layered charts. I can raise this information, but maybe already during validation of sensor attributes rather than when `sensors_to_show` is applied. That is, shall I add it to our `SensorSchema`, as a first implementation of validation for the sensor attributes column?

Reply:
Seems to make sense, if the 2nd level is never to be used (for now)